code/game/objects/structures/bedsheet_bin.dm

```diff
@@ -322,6 +322,17 @@ LINEN BINS
 		amount++
 		to_chat(user, "<span class='notice'>You put [I] in [src].</span>")
 		update_icon()
+	else if(istype(I, /obj/item/wrench) && I.use_tool(src, user, 5, volume=50))

```

Comment:
Use the proper helpers here, default_unfasten_wrench etc.

Reply:
also I.tool_behaviour == TOOL_WRENCH etc reeee.

Reply:
Done for wrench. Screwdriver doesn't have a default proc that I can see (only one for opening panels), so I just changed it to tool behavior.